test/pylib/rest_client.py

```diff
@@ -119,27 +122,29 @@ class ScyllaRESTAPIClient():
     def __init__(self, port: int = 10000):
         self.client = TCPRESTClient(port)
 
-    async def get_host_id(self, server_id: str) -> str:
+    async def get_host_id(self, server_id: IPAddress) -> HostID:
         """Get server id (UUID)"""
         host_uuid = await self.client.get_text("/storage_service/hostid/local", host=server_id)
         assert isinstance(host_uuid, str) and len(host_uuid) > 10, \
                 f"get_host_id: invalid {host_uuid}"
         host_uuid = host_uuid.lstrip('"').rstrip('"')
-        return host_uuid
+        return cast("HostID", host_uuid)
```

Comment:
You're using `HostID(...)` in a similar situation elsewhere, but here you use `cast("HostID", ...)` - wouldn't `HostID(...)` work as well?

Reply:
As above, circular import. This `cast` is understood by type checkers an ignored by the interpreter.

Reply:
Fixed on https://github.com/scylladb/scylladb/commit/0ebda403004c746f8a0019098e9125c160433847